packages/common-server/src/template.ts

```diff
@@ -31,8 +31,16 @@ function copyTemplateProps({
 }: TemplateFunctionProps) {
   const tempNoteProps = _.pick(templateNote, TemplateUtils.TEMPLATE_COPY_PROPS);
   _.forEach(tempNoteProps, (v, k) => {
-    // @ts-ignore
-    targetNote[k] = v;
+    if (k === "custom" && v) {
+      if (targetNote.custom === undefined) targetNote.custom = {};
```

Comment:
if v is defined, this if statement is useless as it will never be true

Reply:
@kevinslin I'm checking if target note has any custom properties. A target note might not have any custom properties defined even when the template note has some custom properties.

Is my assumption is misguided?

Reply:
actually, on second reading, ignore my comment. you're right here